src/map/entities/battleentity.cpp

```diff
@@ -527,7 +527,7 @@ int32 CBattleEntity::addHP(int32 hp)
     hp        = health.hp - cap;
     health.hp = cap;
 
-    // если количество жизней достигает нуля, то сущность умирает
+    // if the number of lives reaches zero, then the entity dies
```

Comment:
this can be removed. "lives" is incorrect and we all know what HP reaching zero means

Reply:
actually the majority of comments in these bricks are uselessly stating the obvious

Reply:
Noted, I'll do a sweep and remove any other Capt. Obvious comments too